src/search/ui/offer.controller.js

```diff
@@ -207,6 +211,29 @@ function OfferController(
     }
   };
 
+  // Confirming event
+  controller.confirmEvent = function () {
+    var labelConfirmed = {name: 'confirmed2020'};
+    controller.labelAdded(labelConfirmed);
+  };
+
+  // Confirming event
+  controller.containsConfirmedTag = function () {
+    var labelConfirmed = {name: 'confirmed2020'};
```

Comment:
For reusability in the coming years the tag itself could be in the appconfig, or I could make the year dynamic to show the current year
@brampauwelyn @bertramakers What are your thoughts?

Reply:
@simonpubliq good idea 👍